docs/creating-fakes.md

```diff
@@ -44,8 +44,10 @@ Examples:
 
 ```csharp
 // Specifying arguments for constructor using expression. This is refactoring friendly!
-// The constructor seen here is never actually invoked. It is an expression and it's purpose
-// is purely to communicate the constructor arguments which will be extracted from it
+// Since the constructor call seen here is an expression, it is not invoked at this point
+// in the code. (In fact no instance of `FooClass` will ever be constructed.) Instead, the

```

Comment:
> In fact no instance of `FooClass` will ever be constructed

I think this could be confusing too. The fake will be an instance of the proxy class, but it's still "an instance of `FooClass`", in the sense that `fake is FooClass` is true.

Reply:
grumble grumble

working on it

Reply:
amended